src/cluster.c

```diff
@@ -571,20 +674,47 @@ int Cluster_GetClusterInfo(RedisModuleCtx *ctx, RedisModuleString **argv, int ar
         }else{
             RedisModule_ReplyWithStringBuffer(ctx, "None", strlen("None"));
         }
+        RedisModule_ReplyWithStringBuffer(ctx, "runid", strlen("runid"));
```

Comment:
Good thing that RG.GETCLUSTERINFO is not documented or you'd have had to update the docs because of this change in the reply :)

Reply:
Prefer not to document it yet, it might change and it is not something that I want users to use.